v3/neson_design/810e.json

```diff
@@ -0,0 +1,308 @@
+        },
+        "2,0",
+        {
+          "c": "#cccccc"
+        },
+        "2,1",
+        "2,2",
+        "2,3",
+        "2,4",
+        "2,5",
+        "2,6",
+        "2,7",
+        "2,8",
+        "2,9",
+        "2,10",
+        "2,11",
+        {
+          "c": "#777777",
+          "w": 2.25
+        },
+        "2,13\n\n\n1,0",
+        {
+          "x": 1.5,
+          "c": "#cccccc"
+        },
+        "2,14",
+        "2,15",
+        "3,16",
+        {
+          "x": 1.5
+        },
+        "2,12\n\n\n1,1"
+      ],
+      [
+        {
+          "c": "#aaaaaa",
+          "w": 1.25
+        },
+        "3,0\n\n\n2,1",
+        {
+          "c": "#cccccc"
+        },
+        "3,1\n\n\n2,1",
+        {
+          "x": 0.25,
+          "c": "#aaaaaa",
+          "w": 2.25
+        },
+        "3,0\n\n\n2,0",
+        {
+          "c": "#cccccc"
+        },
+        "3,2",
+        "3,3",
+        "3,4",
+        "3,5",
+        "3,6",
+        "3,7",
+        "3,8",
+        "3,9",
+        "3,10",
+        "3,11",
+        {
+          "c": "#aaaaaa",
+          "w": 2.75
+        },
+        "3,12\n\n\n3,0",
+        {
+          "x": 1.5,
+          "c": "#cccccc"
+        },
+        "3,15",
+        "4,16",
+        "4,17",
+        {
+          "c": "#777777",
+          "h": 2
+        },
+        "4,15",
+        {
+          "x": 0.5,
+          "c": "#aaaaaa",
+          "w": 1.75
+        },
+        "3,12\n\n\n3,1",
+        {
+          "c": "#777777"
+        },
+        "3,13\n\n\n3,1"
+      ],
+      [
+        {
+          "y": -0.75,
+          "x": 17.75
+        },
+        "3,14"
+      ],
+      [
+        {
+          "y": -0.25,
+          "x": 2.5,
+          "c": "#aaaaaa",
+          "w": 1.25
+        },
+        "4,0\n\n\n4,0",
+        {
+          "w": 1.25
+        },
+        "4,1\n\n\n4,0",
+        {
+          "w": 1.25
+        },
+        "4,2\n\n\n4,0",
+        {
+          "x": 6.75,
+          "w": 1.25
+        },
+        "4,7\n\n\n4,0",
+        "4,8\n\n\n4,0",
+        {
+          "w": 1.25
+        },
+        "4,9\n\n\n4,0",
+        {
+          "x": 3.5,
+          "c": "#cccccc",
+          "w": 2
+        },
+        "4,13\n\n\n5,0"
+      ],
+      [
+        {
+          "y": -0.75,
+          "x": 6.5,
+          "w": 6.25
+        },
+        "4,3",
+        {
+          "x": 4,
+          "c": "#777777"
+        },
+        "4,10",
+        "4,11",
+        "4,12"
+      ],
+      [
+        {
+          "y": -0.25,
+          "x": 20,
+          "c": "#cccccc"
+        },
+        "4,13\n\n\n5,1",
+        "4,14\n\n\n5,1"
+      ],
+      [
+        {
+          "y": -0.75,
+          "x": 3.75,
+          "c": "#aaaaaa"
+        },
+        "4,1\n\n\n4,1",
+        {
+          "w": 1.5
+        },
+        "4,2\n\n\n4,1",
+        {
+          "x": 6.75,
+          "w": 1.5
+        },
+        "4,7\n\n\n4,1",
+        "4,8\n\n\n4,1"
+      ]
+    ]
+  }
+}

```

Comment:
Went on and fixed:

- colors of some keys
- moved the ANSI/ISO selector as a simple ISO enable
- Fixed bottom row HHKB key shifting (the decal was missing)

```suggestion
{
  "name": "810E",
  "vendorId": "0x4E65",
  "productId": "0x810E",
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgblight"
  ],
  "matrix": {
    "rows": 5,
    "cols": 18
  },
  "layouts": {
    "labels": [
      "Split Backspace",
      "ISO",
      "Split Left Shift",
      "Split Right Shift",
      [
        "Bottom Row",
        "Normal",
        "HHKB"
      ],
      "Split 2U 0"
    ],
    "keymap": [
      [
        {
          "x": 15.5
        },
        "0,13\n\n\n0,1",
        {
          "c": "#aaaaaa"
        },
        "0,14\n\n\n0,1"
      ],
      [
        {
          "y": 0.25,
          "x": 2.5,
          "c": "#777777"
        },
        "0,0",
        {
          "c": "#cccccc"
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        "0,9",
        "0,10",
        "0,11",
        "0,12",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "0,14\n\n\n0,0",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "0,15",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "0,16",
        "1,16",
        "0,17",
        "1,17"
      ],
      [
        {
          "x": 2.5,
          "w": 1.5
        },
        "1,0",
        {
          "c": "#cccccc"
        },
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "w": 1.5
        },
        "1,13\n\n\n1,0",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "1,14",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "1,15",
        "2,16",
        "2,17",
        {
          "c": "#aaaaaa",
          "h": 2
        },
        "3,17",
        {
          "x": 2,
          "c": "#777777",
          "w": 1.25,
          "h": 2,
          "w2": 1.5,
          "h2": 1,
          "x2": -0.25
        },
        "2,13\n\n\n1,1"
      ],
      [
        {
          "x": 2.5,
          "c": "#aaaaaa",
          "w": 1.75
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "2,10",
        "2,11",
        {
          "c": "#777777",
          "w": 2.25
        },
        "2,13\n\n\n1,0",
        {
          "x": 1.5,
          "c": "#cccccc"
        },
        "2,14",
        "2,15",
        "3,16",
        {
          "x": 2
        },
        "2,12\n\n\n1,1"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "3,0\n\n\n2,1",
        {
          "c": "#cccccc"
        },
        "3,1\n\n\n2,1",
        {
          "x": 0.25,
          "c": "#aaaaaa",
          "w": 2.25
        },
        "3,0\n\n\n2,0",
        {
          "c": "#cccccc"
        },
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        "3,11",
        {
          "c": "#aaaaaa",
          "w": 2.75
        },
        "3,12\n\n\n3,0",
        {
          "x": 1.5,
          "c": "#cccccc"
        },
        "3,15",
        "4,16",
        "4,17",
        {
          "c": "#aaaaaa",
          "h": 2
        },
        "4,15",
        {
          "x": 0.5,
          "w": 1.75
        },
        "3,12\n\n\n3,1",
        "3,13\n\n\n3,1"
      ],
      [
        {
          "y": -0.75,
          "x": 17.75,
          "c": "#777777"
        },
        "3,14"
      ],
      [
        {
          "y": -0.25,
          "x": 2.5,
          "c": "#aaaaaa",
          "w": 1.25
        },
        "4,0\n\n\n4,0",
        {
          "w": 1.25
        },
        "4,1\n\n\n4,0",
        {
          "w": 1.25
        },
        "4,2\n\n\n4,0",
        {
          "x": 6.75,
          "w": 1.25
        },
        "4,7\n\n\n4,0",
        "4,8\n\n\n4,0",
        {
          "w": 1.25
        },
        "4,9\n\n\n4,0",
        {
          "x": 3.5,
          "c": "#cccccc",
          "w": 2
        },
        "4,13\n\n\n5,0"
      ],
      [
        {
          "y": -0.75,
          "x": 6.5,
          "w": 6.25
        },
        "4,3\n\n\n4,0",
        {
          "x": 4,
          "c": "#777777"
        },
        "4,10",
        "4,11",
        "4,12"
      ],
      [
        {
          "y": -0.25,
          "x": 20,
          "c": "#cccccc"
        },
        "4,13\n\n\n5,1",
        "4,14\n\n\n5,1"
      ],
      [
        {
          "y": -0.75,
          "x": 2.5,
          "c": "#aaaaaa",
          "w": 1.25,
          "d": true
        },
        "4,0\n\n\n4,1",
        "4,1\n\n\n4,1",
        {
          "w": 1.5
        },
        "4,2\n\n\n4,1",
        {
          "x": 6.75,
          "w": 1.5
        },
        "4,7\n\n\n4,1",
        "4,8\n\n\n4,1"
      ],
      [
        {
          "y": -0.75,
          "x": 6.5,
          "c": "#cccccc",
          "w": 6.25
        },
        "4,3\n\n\n4,1"
      ]
    ]
  }
}
```